openedx/core/djangoapps/content/learning_sequences/api/outlines.py

```diff
@@ -309,6 +327,7 @@ def replace_course_outline(course_outline: CourseOutlineData):
         "Replacing CourseOutline for %s (version %s, %d sequences)",
         course_outline.course_key, course_outline.published_version, len(course_outline.sequences)
     )
+    set_custom_attribute('learning_sequences.api.course_id', str(course_outline.course_key))
```

Comment:
Noting you have a large comment for the first of these and nothing for this one. An alternative could be to reuse a one line private method with a docstring.

Aside: What do you think about the idea of introducing `.. custom_attribute_name` and `.. custom_attribute_description` annotations, like we now use for documenting toggles and settings, so you could add inline documentation that automatically gets compiled into a doc like this: https://edx.readthedocs.io/projects/edx-platform-technical/en/latest/featuretoggles.html